addons/library.xbmc.addon/libXBMC_addon.h

```diff
@@ -47,6 +47,8 @@ typedef intptr_t      ssize_t;
 #define ADDON_HELPER_ARCH       "powerpc-osx"
 #elif defined(__arm__)
 #define ADDON_HELPER_ARCH       "arm-osx"
+#elif defined(__aarch64__)
+#define ADDON_HELPER_ARCH       "arm-osx"
```

Comment:
That's what code comments are for... oh wait, hard core devs never add comment :)

Seriously, I'm a very big fan of commenting concepts and ideas in code for when noobs come along :) Keeps the 'oh this is silly, delete', 'oh snap, what broke ?' action down.

This is an area where I'm still thinking on what to do with it and the side effects.